src/Butler/Auth.hs

```diff
@@ -7,20 +7,24 @@ import Butler.Frame
 import Butler.Prelude
 import Lucid.XStatic
 
-publicDisplayApp :: DisplayApplication
-publicDisplayApp = DisplayApplication auth
+newtype PageTitle = PageTitle Text
+    deriving newtype (IsString, ToHtml)
+
+newtype PageDesc = PageDesc Text
+    deriving newtype (IsString)
+
+publicDisplayApp :: PageTitle -> Maybe PageDesc -> DisplayApplication
+publicDisplayApp appTitle appDescM = DisplayApplication auth
   where
-    auth xfiles sessions = guestAuthApp sessions $ htmlMain xfiles "Standalone GUI"
+    auth xfiles sessions = guestAuthApp sessions $ htmlMain xfiles appTitle appDescM
 
-htmlMain :: [XStaticFile] -> Text -> Html () -> Html ()
-htmlMain xfiles title body = do
+htmlMain :: [XStaticFile] -> PageTitle -> Maybe PageDesc -> Html () -> Html ()
+htmlMain xfiles title descM body = do
     doctypehtml_ do
         head_ do
             title_ (toHtml title)
-            meta_ [charset_ "utf-8"]
+            maybe mempty (\(PageDesc desc) -> meta_ [name_ "description", content_ desc]) descM
             meta_ [name_ "viewport", content_ "width=device-width, initial-scale=1.0"]
             xstaticScripts xfiles
             script_ butlerHelpersScript
-
-        with body_ [class_ "font-mono cursor-default bg-stone-100 h-screen"] do
-            body
+            body_ body
```

Comment:
I guess this should be moved up. And let's drop the `body_` prefix so that the caller can set their own class.
```suggestion
        body
```

Reply:
ok